sound/soc/sof/intel/pci-tgl.c

```diff
@@ -65,7 +65,7 @@ static const struct sof_dev_desc ehl_desc = {
 	.default_tplg_path = "intel/sof-tplg",
 	.default_fw_filename = "sof-ehl.ri",
 	.nocodec_tplg_filename = "sof-ehl-nocodec.tplg",
-	.ops = &sof_cnl_ops,
+	.ops = &sof_tgl_ops,
```

Comment:
Ack, this patch is good!

Reply:
there should be a Fixes tag and the commit message should be clearer, e.g.

"
ASoC: SOF: Intel: fix EHL ops

EHL is derived from TGL, not CNL, so we shall use the TGL ops
"